### DISCUSSION TITLE
Web API 的开发工具和平台 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#81] [Comments]
# Web API 的开发工具和平台 | 夜天之书

上一篇文章《Web API 简介》的落脚点是 Web API 的体验。 Web API 作为许多软件的第一道门面，提升其体验的努力从来没有停止过。今天，围绕 Web API 的开发体验和使用体验，已经成长出一个庞大的软件生态。本文以常用的 Web API 开发工具和平台为切入点，介绍当前 Web API 的开发者基础设施。

https://www.tisonkun.org/2023/05/17/api-tools-and-platforms/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

在只有 wget+curl 的时代, 接口也没那么复杂, 一个 word 文档基本可以协作起来,
后来, RESTful 概念重新被发现,
随着前端的爆发,
API 经济引爆,
日常的调试才变成了一件越来越痛苦的事儿,
这才引发了基础工具的发展;


当然, 最惊喜的还是 Swagger ,独立所有语言,
应该说是当年 doctest 概念的衍生,
但是, 非常聪明的完成了一个语言无关的接口文档编译标准,
这一下算是和人工代码历史紧密嵌合在了一起;


不过 AutoGPT 时代, AI 们生成代码时, 这种接口文档反而是阻碍了吧...

**Reply by tisonkun** [reply]
> 不过 AutoGPT 时代, AI 们生成代码时, 这种接口文档反而是阻碍了吧...

这是啥意思？
